fetch.js

```diff
@@ -363,7 +367,7 @@
       }
 
       if ('responseType' in xhr && support.blob) {
-        xhr.responseType = 'blob'
+        xhr.responseType = 'arrayBuffer'
```

Comment:
Whoops!  Good catch.  That was some code I had in to debug what turned out to be an unrelated issue.  I've since removed this change.